src/AssignmentBody.js

```diff
@@ -32,20 +32,19 @@ export default class AssignmentBody extends PureComponent {
         <Heading level="h1" margin="0 0 small">
           {this.props.title}
         </Heading>
-        {this.props.submissionFormats &&
-          this.props.submissionFormats.length > 0 && (
-            <React.Fragment>
-              <div style={{ marginTop: "12px", marginBottom: "12px" }}>
-                <span style={{ fontWeight: "bold" }}>
-                  <Trans>Submitting</Trans>
-                </span>
-                :{" "}
-                <span className="submission-format">
-                  {this.props.submissionFormats}
-                </span>
-              </div>
-            </React.Fragment>
-          )}
+        {this.props.submissionFormats !== null && (
```

Comment:
I'm trying to understand this change. Do you think it's necessary to do?
If necessary, a simple `this.props.submissionFormats &&` won't do the trick?

Reply:
This change is optional (as the assignment body is displayed without it) but it provides the same behavior as Canvas. With an unknown submissionType, `submissionFormats` is an empty string. With this change, the "Submitting" label is rendered with nothing after, as in Canvas. Without this change, the empty string is evaluated as false, and the "Submitting" label does not render in the CCV.

Reply:
For consistency, line 35 could use `!=` like  line 49 instead of `!==`.